src/main/java/net/starlark/java/cmd/Main.java

```diff
@@ -91,23 +112,31 @@ private static void readEvalPrintLoop() {
     // integration with the lexer so that it consumes new
     // lines only until the parse is complete.
```

Comment:
FWIW: we might want to implement a poor man's version of this, which is to keep appending lines to an internal buffer until either a blank line is read, or the buffer can be parsed as a non-empty StarlarkFile. That means you won't have to type newline twice when the input is complete, unless there's a syntax error.

Reply:
Never mind: I see that you did exactly this in a later PR.